app/src/webtrigger/handle-jenkins-request.ts

```diff
@@ -107,8 +110,10 @@ async function handleEvent(
 		return createWebtriggerResponse(400, `invalid event type: ${event.eventType}`);
 	}
 
-	const pipeline: JenkinsPipeline = convertToPipeline(event);
-	await updateJenkinsServerState(jenkinsServerUuid, pipeline, logger);
+	{
```

Comment:
Do we even need these curly braces here?

Reply:
To scope the pipeline variable: it is used only in one place (line below)